core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -356,6 +356,9 @@ void CmpSeabaseDDL::createSeabaseTableLike(
         query += keyClause;
       }
 
+  // send any user CQDs down 
+  Lng32 retCode = sendAllControls(FALSE, FALSE, TRUE);
+
```

Comment:
And to answer your second question: I am not sure. The current behavior of the code is that the CQD is required if doing CREATE TABLE LIKE from sqlci. I think one could make a case for changing that behavior. On the other hand, one might want to prevent casual propagation of nullable primary keys. Which would be an argument for always requiring the CQD. I don't feel strongly either way.